CONTRIBUTING.md

```diff
@@ -207,20 +210,50 @@ easy to reference the pull request where a change was introduced.
      release, and add the github aliases of external contributors to the top of
      the changelog. To lookup an alias from an email, try `bash ./scripts/authors.sh <email>`
    - reset the `CHANGELOG_PENDING.md`
-   - bump versions
+   - bump the appropriate versions in `version.go`
 4. push your changes with prepared release details to `vX.X` (this will trigger the release `vX.X.0`)
 5. merge back to master (don't squash merge!)
 
 #### Minor Release
 
-If there were no breaking changes and you need to create a release nonetheless,
-the procedure is almost exactly like with a new release above.
+Minor releases are done differently from major releases. Minor release pull requests should be labeled with `R:minor` if they are to be included.
+
+1. Checkout the last major release, `vX.X`.
+
+   - `git checkout vX.X`
+
+2. Create a release candidate branch off the most recent major release with your upcoming version specified, `rc1/vX.X.x`, and push the branch.
+
+   - `git checkout -b rc1/vX.X.x`
+   - `git push -u origin rc1/vX.X.x`
+
+3. Create a cherry-picking branch, and make a pull request into the release candidate.
```

Comment:
Can you explain again why do we need cherry-picking? I've only done it a few times while backporting stuff.

Reply:
yes, in this process we are merging all pull requests to master, even breaking ones. In order to do minor releases, we will not cut from master as we have been doing in the past but rather cherry-pick the commits that are meant to go in a minor release.

This is in line with how the sdk is doing things.

Reply:
Okay. Thank you